src/index.ts

```diff
@@ -104,7 +104,11 @@ export class VuexPersistence<S> implements PersistOptions<S> {
     }
 
     this.asyncStorage = options.asyncStorage || false
-    const storageConstructor = this.storage && this.storage.constructor && this.storage.constructor.name.toLowerCase()
+    const storageConstructor =
+      this.storage &&
+        this.storage.constructor &&
+        this.storage.constructor.name &&
```

Comment:
Same check is required around line 149.

Reply:
thank for the catch, added another commit (hope this goes through CI, syntax format/indenting is really strange) 🤞